src/Modules/Finance/index.ts

```diff
@@ -0,0 +1 @@
+export {FinanceLayout as Finance} from './FinanceLayout';
```

Comment:
Could you please clarify why do you export FinanceLayout from index.ts? Why not just to export class FinanceLayout from FinanceLayout.tsx file?

Reply:
We can gather all exported files from the module in one point (index.tsx).
And if we need a few different files from module A in module B, we will use only one import.
Example:
`import {component1, component2} from 'moduleA'`
instead of 
```
import {component1} from 'moduleA/component1'
import {component2} from 'moduleA/component2'
```